massa-execution-worker/src/context.rs

```diff
@@ -713,6 +713,7 @@ impl ExecutionContext {
             block_id: std::mem::take(&mut self.opt_block_id),
             state_changes,
             events: std::mem::take(&mut self.events),
+            gas_cost: None
```

Comment:
it is a bit surprising that this is used only in the case of readonly executions, and kept to none when it's a normal execution